starboard/extension/player_configuration.h

```diff
@@ -32,15 +32,15 @@ typedef struct StarboardExtensionPlayerConfigurationApi {
 
   // The fields below this point were added in version 1 or later.
 
-  // This is used to enforce the underlying starboard player using decode
+  // This is used to ask the underlying starboard player using decode
   // to texture mode to render video frames when it's available, no matter
   // what output mode is passed in SbPlayerCreate(). This function can be
```

Comment:
"no matter what output mode is passed in SbPlayerCreate()", just want to double check if this aligns to the intended usage, as this seems to be risky.